### PR TITLE
=doc #1290 Merge server-side/websocket-support.md

### PR DESCRIPTION
**Pull request by jonas**
References #1290

### CODE REVIEW

**Comment by jonas** [comment]
The diff is best viewed on commit https://github.com/akka/akka-http/pull/1351/commits/22fd7806a0152cb3cfb13e2c8457e936e58a18a1 due to the rename.

**Comment by jonas** [comment]
Yes good point

**Comment by jlprat** [comment]
I have some small comments and some change request for the isText methods and co. Comments are made on the commit directly:
22fd780

File: docs/src/main/paradox/.htaccess [hunk 1]
```diff
@@ -2,3 +2,5 @@
 RedirectMatch 301 ^(.*)/common/de-coding.html$ $1/common/encoding.html
 RedirectMatch 301 ^(.*)/low-level-server-side-api.html$ $1/server-side/low-level-api.html
 RedirectMatch 301 ^(.*)/server-side-https-support.html$ $1/server-side/https-support.html
+# Root at (java|scala)/http to avoid loops.
+RedirectMatch 301 ^(.*)/http/websocket-support.html$$ $1/http/server-side/websocket-support.html
```
Comment (raboof) [inline docs/src/main/paradox/.htaccess]: Should that really be a double `$$`?

Comment (jonas) [reply docs/src/main/paradox/.htaccess]: Ugh, no. Corrected.

**Pull request merged**